### ISSUE TITLE
Feature Request: Specify the feature being enabled in the Pricing grid

### ISSUE THREAD

**Issue by epeicher** [Automattic/wp-calypso#96908]
### What

When upgrading a site following the Install a plugin flow, in the pricing grid, we can read `Upgrade your plan to access this feature and more`. But the user has started the flow by Installing a plugin, so it would be clearer to indicate the specific feature the user is activating, e.g. `Upgrade your plan to be able to Install plugins` or something similar.

### Steps to reproduce 
|| ||
|-------|------|------|
|![Image](https://github.com/user-attachments/assets/857d0dda-9165-4633-9f52-b879cabb2c01) | ![Image](https://github.com/user-attachments/assets/e77e88da-2414-4f64-ac79-178189d4a63c) |![Image](https://github.com/user-attachments/assets/50b8f6cb-9568-497e-8551-eecf92cab143)|

**Note:** This does not seem to be reproducible for all sites, it might be related to an atomic-reverted site

### Why

The user has started the flow by clicking on `Install a plugin` so the Feature they all want to unblock is known

### How

A suggestion could be to use a query param in the URL and adapt the title of the page based on the Feature being installed, which could be `Plugins` or `Themes`

Found in p58i-iNH-p2

**Comment by Robertght** [comment]
Setting a low priority and marking as Triaged on the One Board.
